### PR TITLE
Fix Romania, mixin order matters when mixing in OrthodoxMixin

### PR DESCRIPTION
**Pull request by KidkArolis**
- [x] Tests with a significant number of years to be tested for your calendar.
- [x] Changelog amended with a mention describing your changes. **Note** *Please do NOT change the version number here. It's the project maintainers' duty.*

### CODE REVIEW

**Comment by brunobord** [comment]
I think I see the problem you're solving here, but I'm trying to understand at least one thing: now that the mixin order is swapped, why are the original tests still green?

**Comment by KidkArolis** [comment]
I think its because sometimes Catholic and Orthodox Easter lines up. Worth
double checking though.

**Comment by KidkArolis** [comment]
You can see it below

* in 2019 Easter Sunday is Apr 21 Catholic vs Apr 28 Orthodox
* in 2017 Easter Sunday is Apr 16 in both cases

# 2019

image

image


# 2017

image

image

**Comment by brunobord** [comment]
ok, now I see. If I'm swapping the order of parent classes, the 2019 tests fail.
All you'll have to do is to fix the tiny flake8 issue (a fixup commit or an amended commit + push force would work) and I'll merge this as soon as I can.

File: Changelog.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ## master (unreleased)
 
-Nothing here yet.
+- Fix Romania, make sure it's Easter is calculated using Orthodox calendar
```
Comment (brunobord) [inline Changelog.md]: "its", rather than "it's"

**Comment by KidkArolis** [comment]
Updated

**Comment by brunobord** [comment]
Version 5.2.3 released and pushed on PyPI:

* https://github.com/peopledoc/workalendar/releases/tag/5.2.3
* https://pypi.org/project/workalendar/5.2.3/

**Pull request merged**